WordPress/Classes/System/3DTouch/WP3DTouchShortcutHandler.swift

```diff
@@ -26,15 +26,10 @@ public class WP3DTouchShortcutHandler: NSObject
     static let applicationShortcutUserInfoIconKey = "applicationShortcutUserInfoIconKey"
 
     public func handleShortcutItem(shortcutItem: UIApplicationShortcutItem) -> Bool {
-        var handled = false
-
-        guard let shortCutType = shortcutItem.type as String? else {
-                    return false
-                }
-
         let tabBarController: WPTabBarController = WPTabBarController.sharedInstance()
 
-        switch shortCutType {
+        var handled = false
```

Comment:
Can we remove the `handled` variable? We could just replace

```
handled = true
break
```

with `return true` for each of the case statements, and stick `return false` in the `default` action? Up to you, you might've been trying to avoid multiple returns. :)